src/test/java/core/basesyntax/FruitDaoImplTest.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FruitDaoImplTest {
+    private static final String TEST_FRUIT_NAME = "apple";
+    private static final int TEST_FRUIT_QUANTITY = 10;
+    private FruitDao fruitDao;
+    private Fruit apple;
+
+    @BeforeEach
+    void setUp() {
+        fruitDao = new FruitDaoImpl();
+        apple = new Fruit(TEST_FRUIT_NAME,TEST_FRUIT_QUANTITY);
+        Storage.getInstance().getStorage().clear();
+        fruitDao.add(apple);
+    }
+
+    @Test
+    void addFruit_notNull_success() {
+        assertNotNull(fruitDao.get(TEST_FRUIT_NAME));
+    }
+
+    @Test
+    void getFruit_existingName_success() {
+        assertEquals(apple, fruitDao.get(TEST_FRUIT_NAME));
+    }
```

Comment:
You can check it in one test.

Reply:
Yes, but it`s two separeted tests for me